app/src/main/java/com/hover/stax/domain/use_case/sims/GetLivePresentSimUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.hover.stax.domain.use_case.sims
+import androidx.lifecycle.LiveData
+import com.hover.sdk.sims.SimInfo
+import com.hover.stax.domain.repository.SimRepository
+
+class GetLivePresentSimUseCase(private val simRepository: SimRepository) {
```

Comment:
This can be combined with the `GetPresentSimsUseCase` to form a single class, say `GetPresentSims` since they both do the same thing.